api/app/src/models/medical/mapi-access.ts

```diff
@@ -1,18 +1,13 @@
-import { DataTypes, Sequelize } from "sequelize";
+import { Sequelize } from "sequelize";
 import { BaseModel, defaultModelOptions, ModelSetup } from "../_default";
 
 export class MAPIAccess extends BaseModel<MAPIAccess> {
   static NAME = "mapi_access";
-  declare cxId: string;
 
   static setup: ModelSetup = (sequelize: Sequelize) => {
     MAPIAccess.init(
       {
         ...BaseModel.baseAttributes(),
-        cxId: {
-          type: DataTypes.STRING,
-          primaryKey: true,
-        },
       },
```

Comment:
Why was cxId removed?

Reply:
> rename MAPI access table PK to id

Reply:
That's so we can reuse code/logic based on IDs. An additional pro of this is we have a standard for PKs.